src/Mutate.tsx

```diff
@@ -181,7 +184,14 @@ function Mutate<TError = any, TData = any>(props: MutateProps<TData, TError>) {
   return (
     <RestfulReactConsumer>
       {contextProps => (
-        <RestfulReactProvider {...contextProps} base={`${contextProps.base}${props.path}`}>
+        <RestfulReactProvider
+          {...contextProps}
+          base={
+            props.path.startsWith("/")
```

Comment:
I don't really check how `url.resolve` works, but this should do the stuff for the both cases.